cogs/conversion.py

```diff
@@ -1,19 +1,21 @@
 import re
-from typing import List, Tuple
 
 import discord
 from discord.ext import commands
 
+"""
+Discord cog module for converting imperial units to metric.
+This can be loaded via an extension.
+"""
+
 
 class ConversionCog(commands.Cog):
     """
     A Discord cog for converting height and weight values to metric units.
     """
 
     # Compiled regular expressions as it may improve performance slightly
```

Comment:
This comment is not needed. Add a docstring instead about what the expected input it, and what gets changes.